assets/scss/_styles_project.scss

```diff
@@ -110,28 +110,31 @@
     font-size: 18px;
     margin-bottom: 0;
   }
+
   img {
     margin-top: 1rem;
     width: 20rem;
     max-width: 80%;
   }
+
+  a.external-link:after {
```

Comment:
I feel this block doesn't fit into the PR's responsibility. I'm assuming this is trying to remove the external link icon when a hyperlink exists within the `cncf` section on the homepage?

Reply:
Right and right.

Made a judgement call to fold this into this PR rather than file a separate one.